_contributors-app/debck.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Debasish Sahoo
+description: Open Source Enthusiast and an aspiring Web & Android Developer.
+picUrl: https://user-images.githubusercontent.com/33368759/42726111-cee0efd8-87ac-11e8-9de4-36dfbc8510c8.jpg
+skills: |
+  - **Languages:** Javascript, Java, C, HTML/CSS, Python
+  - **Frameworks:** NodeJs, ExpressJs, Jekyll
+  - **Tools:** GitHub, Git, TravisCI
+
+githubUsername: debck
+speciality: |
+
+  - open Source Development
+
+  - Web Development
+
+  - Android Development
+
+github_url: https://github.com/debck
+linkedIn_url: https://www.linkedin.com/in/debasish2014/
+twitter_url: https://twitter.com/imhere_ck
+
+work_experience: |
+ - Will update soon 
+
+project:
+- |
+    - [Text2Speech Convertor](https://github.com/debck/T-S-C): A simple Java Swing application which converts text written to speech.
+
+- |
+    - [MoVSrcH](https://github.com/debck/MoVSrcH): A simple web application which shows the information about the movie you search for.
+
+- |
+    - [ReWbSp](https://github.com/debck/ReWbSp): A simple nodejs built Web Scraper which scraps the recent top links from Reddit Website
+
+volunteer_experience: |
+ -  Well now I dont have any. But I like to spend most of my time on GitHub, read articles and help anyone in every way possible(Web, Android, Open Source)  
```

Comment:
Well you contribute to OSS so that's volunteer work.

Remove this ~~`Well now I dont have any.`~~